zerver/migrations/0149_realm_allow_community_topic_editing.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.4 on 2017-09-28 22:56
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+from django.db.backends.postgresql_psycopg2.schema import DatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+
+
+def migrate_past_realms_allow_community_topic_editing_false(apps: StateApps, schema_editor: DatabaseSchemaEditor) -> None:
+    Realm = apps.get_model("zerver", "Realm")
+    realms = Realm.objects.all()
+    for realm in realms:
+        realm.allow_community_topic_editing = False
+        realm.save()
```

Comment:
This could be written more cleanly as `Realm.objects.update(allow_community_topic_editing=False)`.

But I think I'm going to temporarily flip the default to False so that it's easier to integrate this partially (without a UI to add it, bugs are less important).